src/character.cpp

```diff
@@ -1009,6 +1009,32 @@ std::vector<item_location> Character::find_ammo( const item &obj, bool empty, in
     return res;
 }
 
+std::vector<item_location> Character::find_reloadables()
+{
+    std::vector<item_location> reloadables;
+
+    visit_items( [this, &reloadables]( item * node ) {
+        if( node->is_holster() ) {
+            return VisitResponse::NEXT;
+        }
+        if( node->is_gun() && !node->magazine_compatible().empty() ) {
+            if( node->magazine_current() == nullptr ||
+                node->ammo_remaining() < node->ammo_capacity() ) {
+                reloadables.push_back( item_location( const_cast<Character &>( *this ), node ) );
```

Comment:
Why `const_cast`? `find_reloadables()` isn't const (maybe it should be).

Reply:
I was trying to make the whole thing const, but gave up, will back this out.